qt/aqt/switch.py

```diff
@@ -91,7 +91,7 @@ def _current_knob_rectangle(self) -> QRectF:
         )
 
     def _paint_knob(self, painter: QPainter) -> None:
-        painter.setBrush(QBrush(theme_manager.qcolor(colors.HIGHLIGHT_BG)))
+        painter.setBrush(QBrush(theme_manager.qcolor(colors.LINK)))
```

Comment:
Does this work for you in both modes? It's the same colour as before in night mode.
Unfortunately, outlining the rounded rectangle looks really bleary.